src/pcapi/admin/custom_views/feature_view.py

```diff
@@ -10,8 +11,15 @@ class FeatureView(BaseAdminView):
     can_edit = True
     column_list = ["name", "description", "isActive"]
     column_labels = dict(name="Nom", description="Description", isActive="Activé")
+    column_searchable_list = ["name"]
     form_columns = ["isActive"]
 
```

Comment:
```suggestion
column_filters = ["isActive"]

```
Un filtre sur `isActive` serait-il bienvenu ?

Reply:
`has_beneficiary_role` du coup ?

Reply:
ajouté